ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/Model.hs

```diff
@@ -368,7 +366,7 @@ advanceCurSlot
   -> Model blk -> Model blk
 advanceCurSlot curSlot m = m { currentSlot = curSlot `max` currentSlot m }
 
-addBlock :: forall blk. (Eq blk, LedgerSupportsProtocol blk)
+addBlock :: forall blk. LedgerSupportsProtocol blk
```

Comment:
We should comment on the `consideredCandidates` local definition that it also includes the currently selected chain, but that it doesn't matter for the result of chain selection